ibis/expr/schema.py

```diff
@@ -235,7 +235,9 @@ def name_at_position(self, i: int) -> str:
         """
         return self.names[i]
 
-    def to_sqlglot(self, dialect: str | sg.Dialect) -> list[sge.ColumnDef]:
+    def to_sqlglot_columns_definition(
```

Comment:
This seems just a tad verbose to me.

```suggestion
    def to_sqlglot_columndefs(
```

or

```suggestion
    def to_sqlglot_columns(
```

or

```suggestion
    def to_sqlglot_coldefs(
```

Reply:
SGTM, let's go with that then!